docs/howto/specify-splits.md

```diff
@@ -0,0 +1,48 @@
+# Howto specify ML data splits
+
+ML datasets may come in [different data splits, intended to be used for different steps of a model building](https://en.wikipedia.org/wiki/Training,_validation,_and_test_data_sets), usually training, validation and testing.
+
+The Croissant format allows for the data to be split arbitrarily into one or multiple splits, which for example allows dataset consumers to load a specific split.
+
+## The split field
+
+The splits definition is done at the `RecordSet` level, with a dedicated field which can be named "split" or anything else, but must have the semantic data type [`wd:Q3985153`](https://www.wikidata.org/wiki/Q3985153). That field data type is generally `sc:Text`, but could theoritically come as other data types, `sc:Integer` for example.
+
+The following artificial example defines the split(s) based on a CSV column:
+
+TODO: example snippet, also with `wd:` context declaration.
```

Comment:
Note for reviewers: I'll need to wait for https://github.com/mlcommons/datasets_format/pull/65 to be merged before I can use the example to dosplay a code snippet.